Modelica/Electrical/QuasiStatic/Polyphase/Blocks/ToSpacePhasor.mo

```diff
@@ -4,6 +4,8 @@ block ToSpacePhasor "Conversion: m phase -> space phasor"
   // import Modelica.ComplexMath.j;
   // import Modelica.ComplexMath.exp;
   // import Modelica.ComplexMath.sum;
```

Comment:
But these can be removed, right?

Reply:
No, as they belong to the commented eq. `c = sqrt(2)/m*sum({u[k]*exp(j*phi[k]) for k in 1:m});`.

Reply:
ah no I see now it's just for the particular equation. Never mind.